project/First-Project/src/main/java/first/project/controller/HelloController.java

```diff
@@ -0,0 +1,21 @@
+package first.project.controller;
+
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class HelloController
+{
+
+    @RequestMapping(value = "/HelloWorld", method = RequestMethod.GET)
```

Comment:
By convention, we should use **hyphenated** style when naming routes, instead of `/HelloWorld` change it to `/hello-world`